### PR TITLE
 added NSEScraper:

### PR DESCRIPTION
**Pull request by ujwalkpl**

### CODE REVIEW

File: Task 1/NationalStockExchange/Solution.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    while d.date() <= end.date():
+        if d.isoweekday() not in excluded:
+            days.append(d)
+        d += datetime.timedelta(days=1)
+    return days
+date_array = []
+
+def strip(days):
+
+
```
Comment (nimishbongale) [inline Task 1/NationalStockExchange/Solution.py:16]: Please remove these blank lines.

File: Task 1/NationalStockExchange/question.md [hunk 2]
```diff
@@ -0,0 +1 @@
+A python script to automatically download Stock exchange data from the <a href="https://www1.nseindia.com/">NSE portal </a>.
```
Comment (nimishbongale) [inline Task 1/NationalStockExchange/question.md:1]: Please add some sample results in markdown format.

File: Task 1/WhatsappScraper/Part1Solution.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+def chats():
+    # name = driver.find_element_by_xpath("//div[@class='_1wjpf _3NFp9 _3FXB1']/span").text
+    name = "scraping"
+    m_arg = '//div[@class="_3zJZ2"]/div'
+    messages = driver.find_elements_by_xpath(m_arg)  
+    print(messages)
+    top_messages = messages[-1*LAST_MESSAGES:]
+    
+
```
Comment (nimishbongale) [inline Task 1/WhatsappScraper/Part1Solution.py:31]: Keep maybe just one blank line as a separator.

File: Task 1/WhatsappScraper/Question.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+This has two parts
+Part1 - Scraper data from Whatsapp web and stores it in a txt file
```
Comment (nimishbongale) [inline Task 1/WhatsappScraper/Question.md:2]: Please add some lines of the output here.

**Review by nimishbongale — CHANGES_REQUESTED** [review]
Good work @ujwalkpl!

Small changes, I'm sure you can do them!

File: Task 1/WhatsappScraper/Part1Solution.py (hunk 3 above)
Comment (ujwalkpl) [reply Task 1/WhatsappScraper/Part1Solution.py:31]: Indeed Bongale sir.

**Pull request closed without merging**